packages/http-client/src/__tests__/ceramic-http-client.test.ts

```diff
@@ -37,6 +37,10 @@ const second = {
 const streamId = new StreamID(0, FAKE_CID_1)
 
 describe('applyCommit', () => {
+  afterEach(() => {
+    delete process.env.OTEL_EXPORTER_PROMETHEUS_HOST;
+    delete process.env.OTEL_EXPORTER_PROMETHEUS_PORT;
```

Comment:
Why are these here?

Reply:
This was an earlier attempt to deal with EADDR_IN_USE test failures, these were actually because of parallelization